src/components/rockets/Reserve/Reserve.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRocket } from '../../../redux/rockets/rockets';
```

Comment:
- [ ] There is a folder naming miss match. You are trying to like this `../../../redux/rockets/rockets`  but the directory structure is actually like this `../../../redux/rockets.js/rockets`